compiler/rustc_error_codes/src/error_codes/E0227.md

```diff
@@ -0,0 +1,33 @@
+This error indicates that the compiler is unable to determine whether there is
```

Comment:
We try to provide a small description of the error first, and then provide more explanations after the erroneous code example. However in this case I'm not sure how to word it... If you have an idea, please go for it, otherwise I'll just approve as is.

Reply:
Yeah, the only reason why I haven't included a small description is because I can't think of one.

Reply:
Then it's fine. I'll approve the PR once the CI passed. Thanks for working on this!